src/commandDetails/coin/leaderboard.ts

```diff
@@ -0,0 +1,75 @@
+import { container, SapphireClient } from '@sapphire/framework';
+import { MessageEmbed } from 'discord.js';
+import {
+  CodeyCommandDetails,
+  CodeyCommandResponseType,
+  getUserIdFromMessage,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse
+} from '../../codeyCommand';
+import {
+  getCurrentCoinLeaderboard,
+  getCoinBalanceByUserId,
+  UserCoinEntry,
+  getUserIdCurrentCoinPosition
+} from '../../components/coin';
+import { EMBED_COLOUR } from '../../utils/embeds';
+
+const getCurrentCoinLeaderboardEmbed = async (
+  client: SapphireClient<boolean>,
+  leaderboard: UserCoinEntry[],
+  currentUserId: string,
+  limit = 10
```

Comment:
still prefer a constant at the top of file, so limit = CONSTANT instead of limit = magic number